zfs/zfs_test.go

```diff
@@ -304,3 +306,55 @@ func TestTryRecvDestroyOrOverwriteEncryptedErr(t *testing.T) {
 	require.NotNil(t, err)
 	assert.EqualError(t, err, strings.TrimSpace(msg))
 }
+
+func TestZFSSendArgsBuild(t *testing.T) {
+	type SendTest struct {
+		conf        ZFSSendArgsUnvalidated
+		argsInclude []string
+		argsExclude []string
+	}
+	var sendTests = map[string]SendTest{
+		"Empty Args":             {conf: ZFSSendArgsUnvalidated{Encrypted: &NilBool{false}}, argsInclude: []string{}, argsExclude: []string{"-w", "-p", "-b"}},
+		"Encrypt":                {conf: ZFSSendArgsUnvalidated{Encrypted: &NilBool{true}}, argsInclude: []string{"-w"}, argsExclude: []string{}},
+		"Raw":                    {conf: ZFSSendArgsUnvalidated{Raw: true}, argsInclude: []string{"-w"}, argsExclude: []string{}},
+		"Send properties":        {conf: ZFSSendArgsUnvalidated{Properties: true}, argsInclude: []string{"-p"}, argsExclude: []string{}},
+		"Send backup properties": {conf: ZFSSendArgsUnvalidated{BackupProperties: true}, argsInclude: []string{"-b"}, argsExclude: []string{}},
+		"Send -b and -p":         {conf: ZFSSendArgsUnvalidated{Properties: true, BackupProperties: true}, argsInclude: []string{"-p", "-b"}, argsExclude: []string{}},
+	}
+
+	for testName, test := range sendTests {
+		t.Run(testName, func(t *testing.T) {
+			args, err := test.conf.buildCommonSendArgs() //fill(encrypted_false))
```

Comment:
leftover comment.
also:
if we don't refactor buildCommonSendArgs, check Encrypted == nil and default to NilBool{false} inside the loop here to save config bloat above.